include/pokemon.h

```diff
@@ -451,44 +451,40 @@ struct MoveInfo
     const u8 *name;
     const u8 *description;
     u16 effect;
-    u8 power;
-    u8 type:5;
-    u8 category:3;
-
+    u16 type:5;
+    u16 category:2;
+    u16 power:9; // up to 511
```

Comment:
Is there a move with more than 255 power?

Reply:
Not in vanilla, but I don't see why we shouldn't allow for that possibility in romhacks. Besides, it's not like we need up to 255 accuracy.

Reply:
Then we don't need to move the extra bit.

Reply:
I can't without reordering things again